TabloidMVC/Views/Account/Index.cshtml

```diff
@@ -0,0 +1,48 @@
+﻿@model IEnumerable<TabloidMVC.Models.UserProfile>
+
+@{
+    ViewData["Title"] = "Index";
+}
+
+<h1>Index</h1>
+
+<p>
+    <a asp-action="Create">Create New</a>
+</p>
+<table class="table">
+    <thead>
+        <tr>
+            <th>
+                @Html.DisplayNameFor(model => model.FullName)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.DisplayName)
+            </th>
+           
+            <th>
+                User Status
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+@foreach (var item in Model) {
+        <tr>
+            <td>
+                @Html.DisplayFor(modelItem => item.FullName)
```

Comment:
Right now, the web page has the column headings as `FullName` and `DisplayName`. Let's add a space between those words just for the readability's sake :) You can add a `[Display(Name="")]` attribute on the model.